### PR TITLE
Add instructions to run specific tests

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: moduleroot/CONTRIBUTING.md.erb [hunk 1]
```diff
@@ -174,6 +174,18 @@ and so on. rspec tests may have the same kind of dependencies as the
 module they are testing. While the module defines in its [Modulefile](./Modulefile),
 rspec tests define them in [.fixtures.yml](./fixtures.yml).
 
+To run specific tests, use the spec test file name and a filter like:
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]: Would it make sense to link to https://relishapp.com/rspec/rspec-core/v/3-9/docs/command-line/example-option

File: moduleroot/CONTRIBUTING.md.erb [hunk 2]
```diff
@@ -174,6 +174,18 @@ and so on. rspec tests may have the same kind of dependencies as the
...
+
+```shell
+bundle exec rspec spec/classes/foreman_spec.rb -e 'should restart passenger'
+```
+
+To run OS specific tests:
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]: https://github.com/mcanevet/rspec-puppet-facts#running-your-tests describes the `SPEC_FACTS_OS` variable. Note you can also leave of things at the end, so `redhat` will test both 7 and 8 if they're in metadata.json.

File: moduleroot/CONTRIBUTING.md.erb (hunk 1 above)
Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: Done

File: moduleroot/CONTRIBUTING.md.erb (hunk 2 above)
Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: nice, this will be very useful.  I added this example to the doc too.

**Comment by sthirugn** [comment]
Ready for re-review

**Comment by sthirugn** [comment]
@ekohl can you merge this too when you get a chance? Thanks

File: moduleroot/CONTRIBUTING.md.erb (hunk 2 above)
Comment (ekohl) [reply moduleroot/CONTRIBUTING.md.erb]: Part of the note was linking to the gem responsible. Knowing we use that can be very useful.

File: moduleroot/CONTRIBUTING.md.erb [hunk 3]
```diff
@@ -174,6 +174,26 @@ and so on. rspec tests may have the same kind of dependencies as the
 module they are testing. While the module defines in its [Modulefile](./Modulefile),
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]: `Modulefile` hasn't been a thing for some time and has been replaced with `metadata.json`.

File: moduleroot/CONTRIBUTING.md.erb (hunk 2 above)
Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: > Part of the note was linking to the gem responsible. Knowing we use that can be very useful.

I don't understand what this means, can you please explain?

File: moduleroot/CONTRIBUTING.md.erb (hunk 3 above)
Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: good point, fixed it.

File: moduleroot/CONTRIBUTING.md.erb (hunk 2 above)
Comment (ekohl) [reply moduleroot/CONTRIBUTING.md.erb:184]: We rely quite heavily on the gem rspec-puppet-facts. A link to that can be useful.

Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb:184]: Thank you for the clarification. I updated links to rspec-puppet-facts now.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**